### PR TITLE
[Fix] Update nav social media links

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

Fixes #181356733

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/config/index.js [hunk 1]
```diff
@@ -176,12 +176,12 @@ export const navigationArgs = {
...
   socialLinks: [
     {
-      href: "https://twitter.com/",
+      href: "https://twitter.com/Code4Africa",
       src: Twitter,
       label: "twitter",
     },
     {
-      href: "https://web.facebook.com/?_rdc=1&_rdr",
+      href: "https://www.facebook.com/CodeForAfrica",
```
Comment (kilemensi) [inline src/config/index.js:184]: QQ @nerdlet , are we still using these in prod? i.e. aren't we pulling them from CMS?

Comment (nerdlet) [reply src/config/index.js:184]: @kilemensi Nope , we shouldn't. Let me remove them.

**Review by KhadijaMahanga — APPROVED** [review]
🚀

**Review by kilemensi — APPROVED** [review]
🚀 

---

Can we also update the CMS for the social media links in the footer to match these ones?

**Pull request merged**